spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/reload/KubernetesClientEventBasedConfigMapChangeDetector.java

```diff
@@ -115,16 +115,6 @@ void inform() {
 			if (enableReloadFiltering) {
 				filter[0] = ConfigReloadProperties.RELOAD_LABEL_FILTER + "=true";
 			}
-
-			// We need to pass an APIClient to the SharedInformerFactory because if we use
```

Comment:
I decided to remove this comment because the default constructor for `SharedInformerFactory` is deprecated anyway